EmptyChronicle/Controller/BlockController.cs

```diff
@@ -71,4 +71,4 @@ public BlockDto GenerateBlockDto(Block block)
             }).ToArray(),
         };
     }
-}
+}
```

Comment:
It seems like unintended changes. You should check your IDE setup whether it removes all newline at the EOF by default. 🤔